spec/time.html

```diff
@@ -228,13 +227,7 @@ <h1>Temporal.Time.prototype.minus ( _temporalDurationLike_ [ , _options_ ] )</h1
         1. Perform ? RequireInternalSlot(_temporalTime_, [[InitializedTemporalTime]]).
         1. Let _duration_ be ? ToLimitedTemporalDuration(_temporalDurationLike_, « »).
         1. Perform ? ToTemporalDisambiguation(_options_).
-        1. Let _hour_ be _temporalTime_.[[Hour]] - _duration_.[[Hour]].
-        1. Let _minute_ be _temporalTime_.[[Minute]] - _duration_.[[Minute]].
-        1. Let _second_ be _temporalTime_.[[Second]] - _duration_.[[Second]].
-        1. Let _millisecond_ be _temporalTime_.[[Millisecond]] - _duration_.[[Millisecond]].
-        1. Let _microsecond_ be _temporalTime_.[[Microsecond]] - _duration_.[[Microsecond]].
-        1. Let _nanosecond_ be _temporalTime_.[[Nanosecond]] - _duration_.[[Nanosecond]].
-        1. Let _result_ be ? BalanceTime(_hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_).
+        1. Let _result_ be ? SubtractTime(_temporalTime_.[[Hour]], _temporalTime_.[[Minute]], _temporalTime_.[[Second]], _temporalTime_.[[Millisecond]], _temporalTime_.[[Microsecond]], _temporalTime_.[[Nanosecond]], _duration_.[[Hours]], _duration_.[[Minutes]], _duration_.[[Seconds]], _duration_.[[Milliseconds]], _duration_.[[Microseconds]], _duration_.[[Nanoseconds]]).
```

Comment:
The short answer is "it was already like that when I found it" :smile: But I do think it makes sense because in order for the operations to be reusable, we might need to use them in contexts where we don't have the instance.

(You would also want to avoid observable property accesses on the instance, but that could equally well be solved by using records instead of instances, so not really a reason to use arguments.)

Reply:
I understand, this is off-topic at best, I will set this as resolved.